spec/unit/http/external_client_spec.rb

```diff
@@ -118,8 +118,8 @@ def do_request(request, options)
       expect(response.code).to eq(200)
     end
 
-    it "sets content-length and content-type for the body" do
```

Comment:
would there be a good way to test that `content-length` is being set in the request, since we're no longer sending it as a header?

Reply:
I left it out because puppetserver's http client may decide to POST using "chunked" transfer encoding in which case the `Content-Length` header would be omitted.